frontend/src/views/camp/Dashboard.vue

```diff
@@ -286,7 +292,9 @@ export default {
             this.filter.progressLabel.length === 0 ||
             this.filter.progressLabel?.includes(
               scheduleEntry.activity().progressLabel?.()._meta.self
-            ))
+            ) ||
+            (scheduleEntry.activity().progressLabel === null &&
+              this.filter.progressLabel?.includes('none')))
```

Comment:
Couldn't you just do the following:
```
            this.filter.progressLabel?.includes(
              scheduleEntry.activity().progressLabel?.()._meta.self ?? 'none'
            ) 
```
means, if ProgressLabel == null, then use dummy-id 'none'

Reply:
Nice, had a moment to wrap my head around